.github/workflows/release-phar.yml

```
@@ -1,46 +1,53 @@
+          path: bin/asd.phar
+
+      - name: Release
+        uses: softprops/action-gh-release@v2
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          name: asd.phar
+
+      - name: Attach artifacts to release
+        uses: actions/upload-release-asset@v1
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
```

actionlint (1.7.4)

property "create_release" is not defined in object type {}

(expression)